docs/source/raft_ann_benchmarks.md

```diff
@@ -398,6 +416,8 @@ The table below contains the possible settings for the `algo` field. Each unique
 By default, the index will be placed in `bench/ann/data/<dataset_name>/index/<name>`. Using `sift-128-euclidean` for the dataset with the `algo` example above, the indexes would be placed in `bench/ann/data/sift-128-euclidean/index/algo_name/param1_val1-param2_val2`.
 
 
+
+
```

Comment:
Straggling blank lines?

Reply:
I did this when I was moving things around because I kept losing my place. I don't think it's hurting anything visually but we can remove it in a follow-on. It's getting late and I want to make sure these changes are ready and deployed for tomorrow.